Daniel-Bennett777/Buildo-API
# added new fixtures and contractor view

##to test## git checkout into test branch
-go into the workorders.py and check for the def my_buildos decorator section.
-go into postman and check with a token that has is_contractor true. 
-copy token id and put it in the authorized header in your postman in the authorized header game.
-`http://localhost:8000/work_orders/my_buildos` is the http request to test. no body needed. If you have any contractor users with accepted jobs currently  will show this response or something similar `[
    {
        "id": 2,
        "customer": {
            "id": 3,
            "bio": "Animals are my passion !",
            "profile_image_url": "https://img.freepik.com/premium-photo/portrait-beautiful-positive-african-american-woman-yellow-studio_93675-137589.jpg?size=626&ext=jpg&ga=GA1.2.1688039518.1694962551&semt=sph",
            "created_on": "2021-09-17",
            "active": true,
            "user_id": 3,
            "state_name": "Tennessee",
            "county_name": "Dickson",
            "is_contractor": false
        },
        "contractor": {
            "id": 4,
            "bio": "I love building things and spending time with my 2 dogs: Charles Barkley and Sasquatch ",
            "profile_image_url": "https://img.freepik.com/free-photo/handsome-young-cheerful-man_171337-1155.jpg?size=626&ext=jpg&ga=GA1.1.1688039518.1694962551&semt=sph",
            "created_on": "2023-11-28",
            "active": true,
            "user_id": 4,
            "state_name": "Tennessee",
            "county_name": "Dickson",
            "is_contractor": true
        },
        "status": {
            "id": 3,
            "status": "in-progress"
        },
        "service_type": "New House Build",
        "state_name": "Tennessee",
        "county_name": "Davidson",
        "date_posted": "12-14-2023",
        "description": "Looking to build a new home, Licensed contractors only",
        "profile_image_url": "https://media.istockphoto.com/id/1363882276/photo/kitchenette.jpg?s=612x612&w=0&k=20&c=u3u7r4fv67KEM7vcV5eT3ma1VmBqfjmMXz4wU-mY-Vw="
    }
]`
